src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.0.props

```diff
@@ -16,13 +16,13 @@
       <Version>4.0.1</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.DotNetHostPolicy">
-      <Version>1.0.10</Version>
+      <Version>1.0.11</Version>
```

Comment:
You'll need to update System.Console here as well for 1.0.0 and 1.1.0 to the corresponding serviced versions on each branch. Sorry I missed that but I realized we where servicing System.Console afterwards. Could you do that on a separate PR?